contrib/docker/docker-compose.yml

```diff
@@ -54,9 +54,13 @@ services:
     image: docker.io/postgres:10-alpine
     # Change that password, of course!
     environment:
+      # we want to create our DB ourselves and this DB gets locked while running the init script
+      - POSTGRES_DB=postgres
       - POSTGRES_USER=synapse
       - POSTGRES_PASSWORD=changeme
     volumes:
+      # DB init script to set LC_COLLATE and LC_CTYPE correctly
+      - ./init-synapse-db.sh:/docker-entrypoint-initdb.d/init-synapse-db.sh
```

Comment:
I believe this is redundant to #6921 ?

Reply:
It does the same thing in a different way. Going to drop this.